clients/mediaconvert.go

```diff
@@ -0,0 +1,155 @@
+						"Audio Selector 1": {
+							DefaultSelection: aws.String("DEFAULT"),
+						},
+					}, 
+					FileInput:      aws.String(input.InputFile), 
+					TimecodeSource: aws.String("ZEROBASED"),
+					VideoSelector:  &mediaconvert.VideoSelector{},
+				},
+			},
+			OutputGroups: []*mediaconvert.OutputGroup{
+				{
+					Name: aws.String("Apple HLS"),
+					OutputGroupSettings: &mediaconvert.OutputGroupSettings{
+						HlsGroupSettings: &mediaconvert.HlsGroupSettings{
+							Destination:      aws.String(input.HLSOutputFile),
+							MinSegmentLength: aws.Int64(0),
+							SegmentLength:    aws.Int64(10),
+						},
+						Type: aws.String("HLS_GROUP_SETTINGS"),
+					},
+					Outputs: []*mediaconvert.Output{
+						{
+							VideoDescription: &mediaconvert.VideoDescription{
+								CodecSettings: &mediaconvert.VideoCodecSettings{
+									Codec: aws.String("H_264"),
+									H264Settings: &mediaconvert.H264Settings{
+										RateControlMode:    aws.String("QVBR"),
+										SceneChangeDetect:  aws.String("TRANSITION_DETECTION"),
+										QualityTuningLevel: aws.String("MULTI_PASS_HQ"),
+										FramerateControl:   aws.String("INITIALIZE_FROM_SOURCE"),
+									}}},
+							AudioDescriptions: []*mediaconvert.AudioDescription{
+								{
+									CodecSettings: &mediaconvert.AudioCodecSettings{
+										Codec: aws.String("AAC"),
+										AacSettings: &mediaconvert.AacSettings{
+											Bitrate:    aws.Int64(96000),
+											CodingMode: aws.String("CODING_MODE_2_0"),
+											SampleRate: aws.Int64(48000),
+										},
+									},
+								},
+							},
+							OutputSettings: &mediaconvert.OutputSettings{
+								HlsSettings: &mediaconvert.HlsSettings{},
+							},
+							ContainerSettings: &mediaconvert.ContainerSettings{
+								Container:    aws.String("M3U8"),
+								M3u8Settings: &mediaconvert.M3u8Settings{},
+							},
+						},
+					},
+					CustomName: aws.String("hls"),
+					AutomatedEncodingSettings: &mediaconvert.AutomatedEncodingSettings{
+						AbrSettings: &mediaconvert.AutomatedAbrSettings{
+							MaxAbrBitrate: aws.Int64(8000000),
+						},
+					},
+				},
+			},
+			TimecodeConfig: &mediaconvert.TimecodeConfig{
+				Source: aws.String("ZEROBASED"),
+			},
+		},
+		Role:  aws.String("arn:aws:iam::055703596054:role/service-role/MediaConvert_Default_Role"),
+		Queue: aws.String("arn:aws:mediaconvert:us-east-1:055703596054:queues/Default"),
+		AccelerationSettings: &mediaconvert.AccelerationSettings{
+			Mode: aws.String("ENABLED"),
+		},
+	})
+
+
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	// print job id
+	fmt.Println(*job.Job.Id)
+
+	// checking if the job is completed (every 3 seconds)
+	for {
+		time.Sleep(3 * time.Second)
+		job, err := svc.GetJob(&mediaconvert.GetJobInput{
+			Id: job.Job.Id,
+		})
+		if err != nil {
+			fmt.Println(err)
+		}
+		fmt.Println(*job.Job.Status)
+		if *job.Job.Status == "COMPLETE" {
+			fmt.Println("Job is complete")
+			break
+		}
+	}
+	return nil
```

Comment:
All these errors need to be returned instead of logged. Ideally you should wrap them [in `fmt.Errorf` errors](https://github.com/livepeer/dms-api/blob/71bcd9d57c35cc7ca33a83d3bcf3b740bd25c0c6/pipeline/mediaconvert.go#L32) for more relevant error messages, but ok to just return the errors as is.

All the other logs should be either removed or replace with structured logging through the `log` package. [example](https://github.com/livepeer/dms-api/blob/71bcd9d57c35cc7ca33a83d3bcf3b740bd25c0c6/pipeline/coordinator.go#L216)